README.md

```diff
@@ -291,6 +284,71 @@ One other tip... This totally works:
 </glamorous.Div>
 ```
 
+#### Overriding component styles
+
+The most common scenario for using props is to override the style of an
+existing component (generated by `glamorous` or not). In this section we'll
+explain how you can use `className`, `css` and composition to override the
+styles of a component. Let's see how that can be done in the examples below.
+
+> Try this out in your browser [here](https://codesandbox.io/s/Kro5369wG)!
```

Comment:
I prefer the last option in your second comment. I don't like to see `{...props}` in a component because of the likelihood of triggering unknown props warnings, not that `{...rest}` wouldn't cause it but at least by destructuring the `props` it might give the hint that it could be sanitized if needed. Added a few more lines in the comment so it (hopefully) makes it clear that `className` is the important bit and that any other props are also supported through `{...rest}`.

Made the other suggested changes as well 👍